velox/common/config/SpillConfig.h

```diff
@@ -103,5 +105,8 @@ struct SpillConfig {
 
   /// CompressionKind when spilling, CompressionKind_NONE means no compression.
   common::CompressionKind compressionKind;
+
+  /// Options passed to velox::FileSystem to create spill WriteFile.
```

Comment:
Can you elaborate a bit more here?

Reply:
I don't know how I can elaborate more on this one. There might be more context needed in Spill, in combination with this comment to understand it. If I put general spill context here it will be a pretty large paragraph.